paper.md

```diff
@@ -38,89 +38,143 @@ affiliations:
     index: 2
   - name: Institut Français de Bioinformatique (IFB)
     index: 3
-  - name: Institute for Biostatistics and Medical Informatics
+  - name: Institute for Biostatistics and Medical Informatics, University of Ljubljana
     index: 4
-date: 23 November 2021
+date: 10 December 2021
 bibliography: paper.bib
 group: \#18 DSW + DAISY
-authors_short: authors?
+authors_short: M. Suchánek, P. Alper, J. Slifka, V. Děd, N. D. Barry, P. Lieby, M. Vidak, and N. Zlender
 git_url: https://github.com/ds-wizard/bh2021-report
 event: BH2021
 ---
 
 # Abstract
 
-This report summarizes our activities and achievements in connecting Data Stewardship Wizard (DSW) and DAISY during the ELIXIR BioHackathon Europe 2021. As a data information system for GDPR compliance, DAISY is very focused on a single goal – gathering all information required for GDPR compliance. On the other hand, DSW is very flexible and can be used even beyond data management planning. We worked on the connection on two fronts. First, we created a new Knowledge Model in DSW to make a data protection impact assessment (DPIA) and develop a related document template. Second, we introduced an integration between projects in DSW and DAISY and querying DAISY data upon document generation. Both of these independent activities brought successful results that were polished and published after the actual BioHackathon. Finally, we provide the related materials as an on-demand training course in the ELIXIR e-Learning Platform.
+This report summarises our activities and achievements in integrating the Data Stewardship Wizard (DSW) and Data Information System (DAISY) tools during the ELIXIR BioHackathon Europe 2021. As a data information system for GDPR compliance, DAISY is focused on a single goal – gathering all information required for GDPR accountability of biomedical research projects. On the other hand, DSW is very flexible and can be used beyond data management planning. We worked on the integration between both tools on two fronts. Firstly, we created a new Knowledge Model in DSW together with a document output template to be able to generate a data protection impact assessment (DPIA). Secondly, we introduced a new integration type between projects in DSW and DAISY that allows the querying of DAISY data upon document generation in DSW. Both of these independent activities brought successful results that were polished and published after the actual BioHackathon. Finally, we provide the related materials as an on-demand training course in the ELIXIR eLearning Platform.
```

Comment:
Just a small comment:
```
results were **polished**
```
 I find this term a bit geeky. Its ok if you want to keep it. Alternatively, it could be replaced by **finalized** or **refined**

Reply:
Thanks!

```suggestion
This report summarises our activities and achievements in integrating the Data Stewardship Wizard (DSW) and Data Information System (DAISY) tools during the ELIXIR BioHackathon Europe 2021. As a data information system for GDPR compliance, DAISY is focused on a single goal – gathering all information required for GDPR accountability of biomedical research projects. On the other hand, DSW is very flexible and can be used beyond data management planning. We worked on the integration between both tools on two fronts. Firstly, we created a new Knowledge Model in DSW together with a document output template to be able to generate a data protection impact assessment (DPIA). Secondly, we introduced a new integration type between projects in DSW and DAISY that allows the querying of DAISY data upon document generation in DSW. Both of these independent activities brought successful results that were refined and published after the actual BioHackathon. Finally, we provide the related materials as an on-demand training course in the ELIXIR eLearning Platform.
```